### DISCUSSION TITLE
What makes an API popular? How can this be quantfied?

### DISCUSSION THREAD

**Discussion by kulnor** [postman-open-technologies/knowledge-base#27] [Announcements]
This recent article from the Postman blog identifies the 5 most popular APIs in the Postman network:
https://blog.postman.com/most-popular-apis-in-the-public-api-network
They currently are: Salesforce, Twitter, WhatsApp, PayPal, and Stripe
A more comprehensive list can be found here: https://www.postman.com/explore/most-popular-apis-this-year
Note that these exists as collections in Postman, the underlying formal OpenAPI specifications are not directly available there. Many other similar article or list can be found out there.

But what makes an API *'popular'*? Number of users? Calls per day? Technical soundness or quality (and if then how is that measured)? And in which community or industry? There can be many definitions of 'popularity' (and the same is true for quality).
What measures can be used to quantify and categorize popularity? Can we infer these from the knowledge base?

Some I can think about as a starting point

Measures:
- number of calls per hour/day,
- number of users
- formally documented (e.g. OpenAPI)
- uptime / availability
- performance / responsiveness
- hosting platform / cloud
- security
- custodian
 
Categorizations:
- industry, sector, domain (private, public, academic)
- Community (perpective)
- Protocol / specification

**Comment by kinlane** [comment]
This is always such an interesting question, yet also difficult area to nail down. I support all of your measures:, and add number of employees and revenue for the company operating the API. 

We should throw together a little schema to help us lock down a baseline, and maybe flesh out for the list of suspects that keeps emerging.

**Comment by kinlane** [comment]
Here is a list I had been curating. I'd like to try and flesh out a schema by applying to these companies we think are popular, then maybe we can find some other lesser known and profile as well - Stripe, Twilio, Twitter, Mastodon, Paypal, Slack, Salesforce, Meta. LinkedIn. Notion, Microsoft Graph, Shutterstock, Snowflake. Plaid, Yodel, Dolby, Docusign, Dropbox, Box, Youtube, GitHub, eBay. New York Times. Spotify

**Comment by kevin-postman** [comment]
I tend to think of an API that is imitated by others as another indicator of it being popular. "Imitation is the sincerest form of flattery". Most discussions I've had around generated documentation for APIs tends to cite Stripe's documentation.. and a lot of APIs have duplicated (or close to it) that form of documentation. Not exactly imitating the API "structure" perhaps.. but I think the gist of what I am saying is understood.
